tests/tls-extra-alert/README.md

```diff
@@ -0,0 +1,12 @@
+# Test Description
+
+This test shows that Suricata generates an additional alert for TLS
+for the given PCAP which shouldn't be there.
+
+## PCAP
+
+Internal.
+
+## Related issues
+
+None so far. State: Trying to establish what's the issue.
```

Comment:
nit: There is a ticket now: https://redmine.openinfosecfoundation.org/issues/7318